corehq/apps/locations/bulk_management.py

```diff
@@ -568,6 +567,19 @@ def _check_unique_location_ids(self):
             for location_id, count in counts if count > 1
         ]
 
+    def _check_new_site_codes_available(self):
+        updated_location_ids = {l.location_id for l in self.all_listed_locations
+                                if l.location_id}
+        # These site codes belong to locations in the db, but not the upload
+        unavailable_site_codes = {l.site_code for l in self.old_collection.locations
+                                  if l.location_id not in updated_location_ids}
+        return [
+            _("Location site_code '{code}' is in use by another location. "
+              "All site_codes must be unique").format(code=l.site_code)
+            for l in self.all_listed_locations
+            if l.site_code in unavailable_site_codes
+        ]
```

Comment:
this would have `All site_codes must be unique` in message for each unavailable site code? is that okay?

Reply:
I think so, because the user needs to know which site codes are problematic.

Reply:
I meant that in case there are two site codes that aren't unique, the messages would be
```
["Location site_code code1 is in use by another location. All site_codes must be unique,
   Location site_code code2 is in use by another location. All site_codes must be unique"]
```

Reply:
That's consistent with the other error messages - they typically repeat the same message for each instance of the error.  I suppose the alternative would be something like "The following site_codes are are in use by another location: code1, code2".  I don't know, not sure I wanna go down that path in this PR.